.github/workflows/master.yml

```
@@ -27,7 +27,50 @@ jobs:
         export PATH="$HOME/miniconda/bin:$PATH"
         source activate test-environment
         ./lint.sh
+  r:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: r-lib/actions/setup-r@v1
+    - name: Query dependencies
```

```suggestion
    # This step dumps the current set of R dependencies and R version into files to be used
    # as a cache key when caching/restoring R dependencies.
    - name: Query dependencies
```

Would be good to add a clarifying comment explaining the purpose of this step

---

Sure. Also, we might want to use the `r-cache` action (https://github.com/r-lib/actions/issues/114) once it's implemented.

---

Yep makes sense!